src/main/webapp/i18n/en/textAssessment.json

```
@@ -51,7 +51,14 @@
                 "dismissFeedbackConfirmation": "Please confirm that you want to delete this feedback!",
                 "feedbackCommentPlaceholder": "Enter your feedback comment here ...",
                 "impactWarning": "This feedback might affect <strong style=\"color:blue\">{{affectedSubmissionsCount}}</strong> other similar submission/s.",
-                "impactWarningTooltip": "The number of times that your feedback will be reused as an automatic suggestion in other submissions that were found to have a similar text block."
+                "impactWarningTooltip": "The number of times that your feedback will be reused as an automatic suggestion in other submissions that were found to have a similar text block.",
+                "automaticFeedbackInformationModal": {
+                    "heading": "Automatic Feedback Information",
+                    "introduction": "A preview of a previously graded submission is shown below. <br />The feedback which is highlighted in <span style='background-color: #d8e8f5'>blue</span> is the one we have reused to generate this automatic feedback.<br/><br/><i>Preview</i>",
```

```suggestion
                    "introduction": "A preview of a previously graded submission is shown below. <br />The <span style='background-color: #d8e8f5'>highlighted feedback</span> was used to generate the automatic feedback.<br/><br/><i>Preview</i>",
```
I would formulate it this way and kept it a bit shorter, but this is only a personal preference, so yours might be better in the current state.